dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -381,7 +381,12 @@ def sample_dqm(self, dqm, time_limit=None, compressed=False, **kwargs):
                 calculates (and describes) the minimum time for your problem.
 
             compressed (binary, optional):
-                Compresses the DQM data when set to True.
+                Compresses the DQM data when set to True. Use if your problem
+                somewhat exceeds the maximum allowed size. Compression tends to
+                be slow and more effective on homogenous data, which in this
+                case means it is more likely to help on DQMs with many identical
+                integer-valued biases than ones with random float-valued biases,
+                for exampel.
```

Comment:
typo

Reply:
Thanks, fixed in the update that deprecates the `compressed` argument